docker-app/qfieldcloud/core/exceptions.py

```diff
@@ -21,7 +22,7 @@ class QFieldCloudException(Exception):
 
     code = "unknown_error"
     message = "QFieldcloud Unknown Error"
-    status_code = None
+    status_code: Optional[int] = None
```

Comment:
```suggestion
    status_code: int | None = None
```

See https://peps.python.org/pep-0604/ .

We use Optional only for python that runs on python 3.9 or older.

Reply:
Indeed, thank you! The operator | is also better for us, type less!